streamdeck_ui/api.py

```diff
@@ -216,6 +219,18 @@ def _on_steam_deck_attached(self, streamdeck_id: str, streamdeck: StreamDeck):
             }
         )
 
+    def set_default_state(self, serial_number: str, deck_type: str):
+        if serial_number not in self.state:
```

Comment:
lets unnest this a bit

if serial_number in self.state:
  return

Reply:
:+1: